scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -13,6 +13,7 @@ public class BuildVersion
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
         public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc2-3{CommitCountString}";
+        public string HostVersion => $"{Major}.{Minor}.1-rc2-{CommitCountString}-00";
```

Comment:
Host evolves independently, as bug fixes happen. CLI is not a nupkg. These two are different components. Host is part of netcore.app. Don't make both the same and do not use cli version for host.